examples/django/django-project/app/templates/header.html

```diff
@@ -0,0 +1,29 @@
+{% load static %}
+
+{% block header %}
+<header>
+  <div>
+    <a href="#main-content" class="button focus-only skip-to-content"
+      >Ga direct naar inhoud</a
+    >
+    <nav
+      data-open-label="Menu"
+      data-close-label="Sluit menu"
+      data-media="(min-width: 30rem)"
+      aria-label="Hoofdnavigatie"
+      class="collapsible"
+    >
+      <a href="/" class="logo">
+        <img src="{% static 'img/logo.svg' %}" alt="Logo" />
```

Comment:
The contrast of the logo (#fff on #fff) is... uh... insufficient... :sweat_smile: